modules/users/client/controllers/settings/change-password.client.controller.js

```diff
@@ -33,11 +33,13 @@
       // If successful show success message and clear form
       $scope.$broadcast('show-errors-reset', 'vm.passwordForm');
       vm.success = true;
```

Comment:
Since these aren't used by the view any longer, we should remove them from the controller.

@sujeethk I know you mentioned having issues with the client-side tests that relied on these `vm` fields, but they you should be able to resolve them. Let me know if you have trouble, and I can take a look.